### DISCUSSION TITLE
how to configure --new-from rev to consider running linter on whole branch compare to master

### DISCUSSION THREAD

**Discussion by yash97** [golangci/golangci-lint#1850] [Q&A] [answered]
Basically how to run linter on delta of master to feature branch

**Comment by ldez** [comment, marked as answer]
Hello,

Take a look here: https://golangci-lint.run/usage/faq/#how-to-integrate-golangci-lint-into-large-project-with-thousands-of-issues

```
--new-from-rev=master
```

**Reply by yash97** [reply]
so if i do this i get this error Can't process result by diff processor: can't prepare diff by revgrep: could not read git repo: error executing git diff \"master\" \"\": exit status 128"

**Reply by ldez** [reply]
have you checked that you have the branch `master` on your CI?
